build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        kotlin_version = '1.3.50'
+        kotlin_version = '1.4.30-RC'
```

Comment:
Seems like latest version is 1.4.30 already (https://github.com/JetBrains/kotlin/releases/tag/v1.4.30) - without RC
I am ok to update kotlin version as long as all tests pass and there is no issue in normal execution, but I think we should use 1.4.30 (without RC)

Reply:
Changed to  1.4.30 @tibagni